incubator/appdaemon/0.1.0/test_values.yaml

```diff
@@ -0,0 +1,47 @@
+image:
+  repository: ghcr.io/truecharts/appdaemon
+  pullPolicy: IfNotPresent
+  tag: "v4.0.8"
+
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 568
+  runAsGroup: 568
+  fsGroup: 568
+
+timezone: "America/Chicago"
+
+env:
+  LATITUDE: 46
+  LONGITUDE: -94
+  ELEVATION: 1217
+  DASH_URL: "http://0.0.0.0:5050"
+
+
+services:
+  main:
+    port:
+      port: 5050
+  tcp:
+    enabled: true
+    type: ClusterIP
+    port:
+      port: 51050
+      protocol: TCP
+      targetPort: 51050
+ingress:
+  main:
+    enabled: false
+
+persistence:
+  config:
+    enabled: true
+    mountPath: "/config"
+    emptyDir: true
+    accessMode: ReadWriteOnce
+    size: 1Gi
+    storageClass: ""
```

Comment:
You again didn't work correctly:

- FIRST you change the test_values.yaml. 
- You test that
- THEN you adapt questions.yaml

You are making it hard on yourself by not following that order.

Reply:
Nasty habit! Not used to having any kind of CI/Testing environment.

Reply:
I understand the habbit, the thing is: Manual testing takes quite some time and makes it hard for us to review your changes.

By first sending in the test_values.yaml, you can often save yourself a manual test-run when easy-to-spot changes are spotted by the CI ;-)

Reply:
How would I go about testing the custom storage since since I don't have a host path to mount in CI?

Reply:
emptyDir and/or /tmp